frontend/question.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>question</title>
+</head>
+<body>
+	<div id="logo">
+		<a href="index.html">
+			<img src="res/devcenter-logo.svg">
+		</a>
+		
+	</div>
+	<div id="name">
+		<h2>Coding101</h2>
+	</div>
+	
+	<form>
+		<div id="quiz">
+			<label>I am a Secondary School Graduate</label>
+		
+			<div class="quest-1">
+				<li>
+
+				    <h3>What do you prefer as a hobby/ what do you do in your free time</h3>
+				    <hr>
+				    
+				    <div>
+				        <input type="checkbox" name="question-1-answers" id="question-1-answers-A" value="A" />
```

Comment:
the checkbox should be radio button as you can only pick one answer per question